sectionproperties/post/post.py

```diff
@@ -189,17 +189,11 @@ def print_results(cross_section, fmt):
 
     j = cross_section.get_j()
     if j is not None:
-        if list(set(cross_section.materials)) != [DEFAULT_MATERIAL]:
-            print("E.J\t = {:>{fmt}}".format(j, fmt=fmt))
-        else:
-            print("J\t = {:>{fmt}}".format(j, fmt=fmt))
+        print(prefix + "J\t = {:>{fmt}}".format(j, fmt=fmt))
```

Comment:
I like this `prefix` idea! Nice approach!

Reply:
Does this PR obviate #120? (Sorry, it was a good opportunity to use a $5 word XD)

Reply:
Yep, sorry got that a bit muddled!